ADVANCED.md

```diff
@@ -538,35 +538,35 @@ display 24-bit colors.
 
 ```sh
 # junegunn/seoul256.vim (dark)
-export FZF_DEFAULT_OPTS='--color=bg+:#3F3F3F,bg:#4B4B4B,border:#6B6B6B,spinner:#98BC99,hl:#719872,fg:#D9D9D9,header:#719872,info:#BDBB72,pointer:#E12672,marker:#E17899,fg+:#D9D9D9,preview-bg:#3F3F3F,prompt:#98BEDE,hl+:#98BC99'
+export FZF_DEFAULT_OPTS='--color=border:#6b6b6b,fg+:#d9d9d9,preview-bg:#3f3f3f,info:#bdbb72,spinner:#98bc99,prompt:#98bede,bg+:#3f3f3f,fg:#d9d9d9,disabled:#6b6b6b,header:#719872,marker:#e17899,hl+:#98bc99,query:#719872,hl:#719872,pointer:#e12672'
 ```
 
 ![seoul256](https://user-images.githubusercontent.com/700826/113475011-2c192d80-94ae-11eb-9d17-1e5867bae01f.png)
 
 ```sh
 # junegunn/seoul256.vim (light)
-export FZF_DEFAULT_OPTS='--color=bg+:#D9D9D9,bg:#E1E1E1,border:#C8C8C8,spinner:#719899,hl:#719872,fg:#616161,header:#719872,info:#727100,pointer:#E12672,marker:#E17899,fg+:#616161,preview-bg:#D9D9D9,prompt:#0099BD,hl+:#719899'
+export FZF_DEFAULT_OPTS='--color=border:#c8c8c8,fg+:#616161,preview-bg:#d9d9d9,info:#727100,spinner:#719899,prompt:#0099bd,bg+:#d9d9d9,fg:#616161,disabled:#c8c8c8,header:#719872,marker:#e17899,hl+:#719899,query:#719872,hl:#719872,pointer:#e12672'
 ```
 
 ![seoul256-light](https://user-images.githubusercontent.com/700826/113475022-389d8600-94ae-11eb-905f-0939dd535837.png)
 
 ```sh
 # morhetz/gruvbox
-export FZF_DEFAULT_OPTS='--color=bg+:#3c3836,bg:#32302f,spinner:#fb4934,hl:#928374,fg:#ebdbb2,header:#928374,info:#8ec07c,pointer:#fb4934,marker:#fb4934,fg+:#ebdbb2,prompt:#fb4934,hl+:#fb4934'
+export FZF_DEFAULT_OPTS='--color=fg+:#3c3836,preview-bg:#d5c4a1,info:#427b58,spinner:#9d0006,prompt:#9d0006,fg:#3c3836,header:#928374,marker:#9d0006,hl+:#9d0006,query:#928374,hl:#928374,pointer:#9d0006'
```

Comment:
The colors doesn't look right.

Before:

<img width="1728" alt="image" src="https://github.com/junegunn/fzf/assets/700826/b68b9111-f2be-4498-b8a3-998b52ac7479">

After:

<img width="1728" alt="image" src="https://github.com/junegunn/fzf/assets/700826/8401cb77-b464-436e-9185-478f87f71da5">

Reply:
Which theme were you trying?